mne/io/fiff/tests/test_raw_fiff.py

```diff
@@ -1318,6 +1318,15 @@ def test_crop():
     assert raw.n_times - 1 == raw3.n_times
 
 
+@testing.requires_testing_data
+def test_resample():
```

Comment:
See [failing CI](https://dev.azure.com/mne-tools/mne-python/_build/results?buildId=32275&view=logs&jobId=c1ec3add-0fcd-52ea-530a-328457bbfb2e&j=c1ec3add-0fcd-52ea-530a-328457bbfb2e&t=056d958c-96b0-5ff4-dcb2-798466eed616), there is currently a test with the same name
```suggestion
def test_resample_with_events():
```
and you'll need to `from mne import events_from_annotations` or similar at the top